packages/fxa-payments-server/src/components/fields/index.tsx

```diff
@@ -252,7 +247,22 @@ export const defaultStripeElementValidator: OnValidateFunction = (
   };
 };
 
-export const StripeElement = (props: StripeElementProps) => {
+type StripeElementWrapperProps = {
+  onValidate?: OnValidateFunction;
+} & FieldProps & {
+    component: any;
+    getString: Function | undefined;
+  };
```

Comment:
`getString?: Function` instead?

Any reason to split up `{ onValidate? }` and `{ component; getString }`?